enhancements/upgrade-bundle.md

```diff
@@ -0,0 +1,138 @@
+---
+title: upgrade-bundle
+authors:
+  - "@rashmigottipati"
+reviewers:
+  - "@joelanford"
+  - "@estroz"
+  - "@shawn-hurley"
+  - "@jmrodri"
+approvers:
+  - TBD
+creation-date: 2020-06-11
+last-updated: 2020-06-11
+
+---
+
+# Upgrade Bundle
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+1. How do we handle multiple subscriptions? 
+2. Should we explicitly pass new version number as a flag or can we get it from bundle image?
+    * Based on discussions, can extract the version number from bundle image. 
+3. How do we verify that the upgrade is successful 
+4. Is it valid to assume that no previous version of the operator is already running? 
```

Comment:
I think the scope of this ep should be narrowed to "implement `run bundle-upgrade` for operators that don't exist in a cluster" since the "existing operator" scenario is different implementation-wise.